index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Capstone Project x01</title>
+</head>
+<body>
+        <div class="container">
+          <div class="loggings desktop">
+            <a href="#"><img src="./assets/desktop/facebook.png" alt="facebook"></a>
+            <a href="#"><img src="./assets/desktop/twitter.png" alt="twitter"></a>
+            <a href="#"><p>Logout</p></a>
+            <a href="#"><p>Mypage</p></a>
+            <a href="#"><p>Korean</p></a>
+          </div>
+          <header>
+            <img class="hambuger" src="./assets/mobile//hamburger.png" alt="harmbuger icon"/>
+            <nav class="navigator toggle-ON">
+              <button class="exit-btn">X</button>
+              <a href="index.html"><img class="desktop" src="./assets/mobile/logo.svg" alt="logo"></a>
+              <ul>
+                <li><a href="about.html">About</a></li>
```

Comment:
[required] The previous reviewer talked about font colour. Kindly change the font colour for the `About Button` to fit the template. The aim behind ensuring you follow is to train you on following client requirements in the future. Once you fix that your project is due for approval

Reply:
Honestly, this could serve as optional feedback. Worked so hard on this project, is annoying to see it keep getting rejected over some minor issues.

Alright, I will adjust that. I hope another reviewer after you will finally approve it.